.github/workflows/bump_version_release.yaml

```
@@ -36,62 +36,72 @@ on:
 
 run-name: Deploying ${{ inputs.repo }} ${{ inputs.branch }}
 
+permissions: {}
+
 jobs:
   release-version-bump:
     runs-on: ubuntu-latest
+    # permissions required for secrets.BUMP_N_RELEASE token:
+    #   contents: write # to `git push` and create releases
+    #   pull-requests: read # to read PR info when regenerating the changelog
     steps:
-      - name: checkout ${{ inputs.repo }}
+      - name: Checkout ${{ inputs.repo }}
         uses: actions/checkout@v5
         with:
+          persist-credentials: true # needed for `git push`
           repository: nRF24/${{ inputs.repo }}
           ref: ${{ inputs.branch }}
           fetch-depth: 0
           path: ${{ inputs.repo }}
           token: ${{ secrets.BUMP_N_RELEASE }}
 
-      - name: checkout org repo
+      - name: Checkout org repo
         uses: actions/checkout@v5
         with:
+          persist-credentials: false
           path: org-repo
 
       - uses: actions/setup-python@v6
         with:
-          # here we need v3.10+
-          python-version: 3.x
-
-      - run: rustup update --no-self-update
-
-      - name: Install cargo-binstall
-        uses: cargo-bins/cargo-binstall@main
+          python-version: ">=3.10"
 
       - name: Install git-cliff
-        run: cargo binstall -y git-cliff --install-path org-repo
+        run: pip install -y org-repo/.github/requirements.txt
```

fix short arg name used here
```suggestion
        run: pip install -r org-repo/.github/requirements.txt
```